simulation/README.md

```diff
@@ -102,7 +102,7 @@ disadvantages:
 - transparent
 - vain
 strategies:
-  action: AlwaysAttackStrategy
+  action: AlwaysAttackActionStrategy
```

Comment:
I'm guessing this was renamed at some point, which caused the README to become outdated.